app/Http/ViewComposers/ProfileComposer.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\ViewComposers;
+
+use Illuminate\View\View;
+// use App\Repositories\Contracts\OptionRepository;
+use App\Eloquent\Option;
+
+class ProfileComposer
+{
+
+    /**
+     * Create a new profile composer.
+     *
+     * @return  void
+     */
+    protected $option;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.FunctionDeclarationArgumentSpacing.SpacingBetween</br>**Severity**: error</br>**File**: app/Http/ViewComposers/ProfileComposer.php</br>**Line**: 19</br>Expected 0 spaces between parenthesis of function declaration; 0 found